test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
They were all the same PR, now I have the PR #12023, I've replaced them with foo, bar, baz, and referenced the PR in the comment above.